components/Dashboard/Radio/RadioGroup.tsx

```diff
@@ -12,24 +12,41 @@ type Props = {
   value?: string
   options: Option[]
   onChange?: (value: string | string[]) => void
+  renderOption?: (option: Option, selectedOption: string) => JSX.Element
+  radioGroupClassName?: string
 }
 
 export const RadioGroup = forwardRef(
   (props: Props, ref: ForwardedRef<HTMLDivElement>) => {
-    const { name, value = '', options, onChange } = props
+    const {
+      name,
+      value = '',
+      options,
+      onChange,
+      renderOption,
+      radioGroupClassName,
+    } = props
 
     return (
       <RG
         value={value}
         name={name}
         onChange={onChange}
-        className="flex flex-wrap w-full text-sm"
+        className={
+          radioGroupClassName
+            ? radioGroupClassName
+            : 'flex flex-wrap w-full text-sm'
+        }
```

Comment:
I want to hear your thought @ngolapnguyen, do you think it's okay to do like this?